packages/server/test/notificationsHelper.ts

```diff
@@ -0,0 +1,173 @@
+import { getQueue, NotificationJobResult } from '@/modules/notifications/services/queue'
+import { EventEmitter } from 'events'
+import { CompletedEventCallback, FailedEventCallback, JobId } from 'bull'
+import { pick } from 'lodash'
+
+type AckEvent = {
+  result?: NotificationJobResult
+  err?: Error
+  jobId: JobId
+}
+
+const NEW_ACK_EVENT = 'new-ack'
+
+export async function buildNotificationsStateTracker() {
```

Comment:
Utility you can use in tests to await when certain notifications finish executing